src/Entity/AbstractEntity.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of aakb/jira_economics.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Blameable\Traits\BlameableEntity;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+/**
+ * Class AbstractEntity.
+ */
+abstract class AbstractEntity
+{
+    use BlameableEntity;
+    use TimestampableEntity;
+
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __toString()
+    {
+        return (string) $this->id;
+    }
+}
```

Comment:
Well, this seems like lazy programmer's solution … `static::class . $this-id` may be a better value, but I think we should let the actual entities implement `__toString`.

Reply:
Removed.